src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -0,0 +1,135 @@
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.funcotator.dataSources.gencode.GencodeFuncotationFactory;
+import org.broadinstitute.hellbender.tools.funcotator.vcfOutput.VcfOutputRenderer;
+import org.broadinstitute.hellbender.utils.codecs.GENCODE.*;
+
+import java.io.File;
+import java.util.*;
+
+/**
+ * Funcotator (FUNCtional annOTATOR) performs functional analysis on given variants
+ * and reports output in a specified output file.
+ *
+ * This tool is the GATK analog of the Oncotator.
+ *
+ * Created by jonn on 8/22/17.
+ */
+@CommandLineProgramProperties(
+        summary = "Create functional annotations on given variants cross-referenced by a given database.\n" +
+                "A GATK version of the Oncotator.",
+        oneLineSummary = "(Experimental) Functional Annotator",
+        programGroup = VariantProgramGroup.class
+)
+@DocumentedFeature
+@BetaFeature
+public class Funcotator extends VariantWalker {
+
+    public static final String GTF_FILE_ARG_LONG_NAME = "gtfFile";
+    public static final String GTF_FILE_ARG_SHORT_NAME = "gtf";
+    public static final String GENCODE_FASTA_ARG_NAME = "fasta";
+
+    //==================================================================================================================
+
+    @Argument(
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            fullName  = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            doc = "Output VCF File.")
+    protected File outputFile;
+
+    @Argument(
+            shortName = GENCODE_FASTA_ARG_NAME,
+            fullName  = GENCODE_FASTA_ARG_NAME,
+            doc = "GENCODE Transcript FASTA File.")
+    protected File gencodeTranscriptFastaFile;
+
+    @Argument(
+            fullName = GTF_FILE_ARG_LONG_NAME,
+            shortName = GTF_FILE_ARG_SHORT_NAME,
+            doc = "A GENCODE GTF file containing annotated genes."
+    )
+    private FeatureInput<GencodeGtfFeature> gtfVariants;
+
+    //==================================================================================================================
+
+    private OutputRenderer outputRenderer;
+    private final List<DataSourceFuncotationFactory> dataSourceFactories = new ArrayList<>();
+
+    //==================================================================================================================
+
+    @Override
+    public boolean requiresReference() {
+        return true;
+    }
+
+    @Override
+    public void onTraversalStart() {
+        logger.info("Beginning traversal");
```

Comment:
I think the VariantWalker will output a message like this for you (progress meter).

Reply:
Fixed.